backend/core/src/seeder/seed.ts

```diff
@@ -331,9 +328,8 @@ async function seed() {
   const mfaRoles: UserRoles = { user: mfaUser, isPartner: false, isAdmin: true }
   await rolesRepo.save(mfaRoles)
   await userService.confirm({ token: mfaUser.confirmationToken })
-
-  const alamedaJurisdiction = jurisdictions.filter((j) => j.name == CountyCode.alameda)[0]
-  const alamedaAdmin = await userService.createPublicUser(
+  const bayAreaJurisdiction = jurisdictions.filter((j) => j.name == CountyCode.bay_area)[0]
+  const bayAreaAdmin = await userService.createPublicUser(
     plainToClass(UserCreateDto, {
       email: "alameda-admin@example.com",
```

Comment:
Why is bayAreaAdmin using values with the alameda prefix?

Reply:
yes that is outside of the scope of this CL (see description) if I were to update everything it would be too much for a single CL, and I still need to think about what seed data we specifically need. 

This CL simply changes the seed scripts to use the new Bay Area `Jurisdiction` (like the row in the `Jurisdiction` table not the concept of the Bay Area jurisdiction)  and makes the Bay Area `Jurisdiction` the only row in theat table 

"jurisdiction" being an overloaded term is very annoying!